src/main/kotlin/gg/essential/elementa/components/ScrollComponent.kt

```diff
@@ -32,7 +32,8 @@ class ScrollComponent constructor(
     private val pixelsPerScroll: Float = 15f,
     private val scrollAcceleration: Float = 1.0f,
     customScissorBoundingBox: UIComponent? = null,
-    private val passthroughScroll: Boolean = true
+    private val passthroughScroll: Boolean = true,
+    private val limitScrollBarGripHeight: Boolean = false,
```

Comment:
Do we or anyone really ever want this to not be `true` (provided vertical scrolling is enabled)?
One might disagree about what the "correct" minimum is, but I don't think anyone will ever want it to actually go down to a single pixel.

If we're concerned about backwards compatibility, we should put the change behind a new ElementaVersion.

Reply:
>  One might disagree about what the "correct" minimum is, but I don't think anyone will ever want it to actually go down to a single pixel.

It was mostly because of backwards compatibility, the Elementa version probably makes more sense.